### ISSUE TITLE
Documentation Change - Profile Components

### ISSUE THREAD

**Issue by nine13tech** [Bungie-net/api#813]
In the Profile components documentation, I'd like to see the documentation change to reflect the results from the API.
For instance: Kiosks in the docs is really `characterKiosks` in the API.
This can be confusing to some people (like myself) when trying to make sense of all the data.

https://bungie-net.github.io/multi/schema_Destiny-DestinyComponentType.html#schema_Destiny-DestinyComponentType

**Comment by nine13tech** [comment]
I think maybe we are going in the wrong direction... possibly and I would like to refine this ask a bit.
Specifically, on this page:
https://bungie-net.github.io/multi/schema_Destiny-DestinyComponentType.html#schema_Destiny-DestinyComponentType
current:
`Profiles: 100
Profiles is the most basic component, only relevant when calling GetProfile. This returns basic information about the profile, which is almost nothing: a list of characterIds, some information about the last time you logged in, and that most sobering statistic: how long you've played.`
proposed:
`Profiles: 100
datapoint name: profiles
Profiles is the most basic component, only relevant when calling GetProfile. This returns basic information about the profile, which is almost nothing: a list of characterIds, some information about the last time you logged in, and that most sobering statistic: how long you've played.`

Just a thought.
or maybe add linking to the schema return data so that it is readily findable from these types of pages? Something to make it more obvious where we are to look for this data...!

**Comment by vthornheart-bng** [comment]
Yeah, I could definitely picture us adding links back to the actual properties that get populated from DestinyComponentType, I dig it!
